### ISSUE TITLE
overbid after first XX

### ISSUE THREAD

**Issue by ThePokerDude** [EdwardPiwowar/BBA#275]
I think wet's 2s is not 11-15 but more like 11-13-.
It shows weekness.
Thus east should pass

3A92FF0FEAA1CD4979D4FFE61B91
![image](https://github.com/EdwardPiwowar/BBA/assets/33804161/a5d940ff-daf1-407d-a856-e96134e2750f)

**Comment by ThorvaldAagaard** [comment]
Agree, I think the theme is described in one of the earlier issues (still pending)

**Comment by ThePokerDude** [comment]
better in 8627. But East should pass instead of 3s
![image](https://github.com/user-attachments/assets/e86a2f4e-a2b7-442a-a353-3c8a9373e32d)
